kioku.cabal

```diff
@@ -2,7 +2,7 @@
 --  see http://haskell.org/cabal/users-guide/
 
 name:                kioku
-version:             0.1.2.0
+version:             0.1.2.1
```

Comment:
This change seems in line with PVP guidelines but I was surprised that they didn't speak to compiler version. It *seems* like there'd be a reason that changing compiler version would warrant a greater bump in version than just the `C` component.

Reply:
The reason that the PVP guidelines don't talk about compiler version that the compiler is not part of the package. The user of the package brings the compiler with them.

It's certainly possible that supporting a newer compiler version could require breaking API changes, in which case `A.B` would need to be bumped.